code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -2090,6 +2090,11 @@
 	glass_desc = "You've really hit rock bottom now... your liver packed its bags and left last night."
 	taste_tag = list(TASTE_BITTER,TASTE_DRY)
 
+/datum/reagent/alcohol/hooch/affect_ingest(mob/living/carbon/M, alien, effect_multiplier)
+	. = ..()
+	if(M.stats.getPerk(PERK_ALCOHOLIC))
+		M.add_chemical_effect(CE_PAINKILLER, 15) //Acts as Inaprovaline painkiller if alcoholic
```

Comment:
Alcohol already provides painkiller effect, depending on it's strength.
With formula of `max(35 - (strength / 2), 1))` and strength of hooch of 1, we'll get 34.5 painkiller effect.
This adds 15 on top, to a whooping 49.4, same as tramadol.

Reply:
I see, I didn't realize that. Changed effect to heal OxyLoss at -0.3, half strength of Tricord. I didn't have any other ideas for what the effect could be, so this is kinda a placeholder until someone gets an actually good idea.